static/app.webmanifest

```diff
@@ -8,6 +8,11 @@
     "src": "images/icon.png",
     "sizes": "256x256",
     "type": "image/png"
+  }, {
+    "src": "images/icon-2x.png",
+    "sizes": "512x512",
+    "type": "image/png"
   }],
-  "background_color": "#5d9bc7"
+  "background_color": "#5d9bc7",
+  "theme_color": "#75AACF"
```

Comment:
Nit: I'd suggest the theme_color is darker than background_color rather than lighter, e.g. #5288af